app/views/courses/_cohort_courses.html.erb

```diff
@@ -17,7 +17,8 @@
     </div>
     <div class="cohort-select">
       <select class="cohorts">
-        <option selected disabled><%= t("cohort.view_other") %></option>
+        <option selected disabled>
+        <%= Features.wiki_ed? ? t("courses.view_other") : t("courses_generic.view_other") %></option>
```

Comment:
This pattern shows up over and over. This is what we should use the helper method for. We could change the signature to `course_i18n(message_key)` and handle all the conditionals there. That would make each of these cases be simply `course_i18n('view_other')` and so on.